canonical-web-and-design/maas-ui
# Create settings configuration general form

## Done
- Created general component. Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1379
- Displayed initial values in general form. Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/138.
- Implemented saving general form. Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1382

## QA
- Go to `/settings/configuration#general`
- See that there is a field for "MAAS name" and a checkbox to enable analytics
- Remove the value from the "MAAS name" field and see that the "Save" button is disabled
- Change the values of both the form fields
- Submit the form
- See that as you are saving the "Save" button indicates that it is saving
- Reload the page and see that the values you changed have persisted
